### PR TITLE
Change erb to slim

### PR DESCRIPTION
**Pull request by marcelo140**

### CODE REVIEW

File: app/assets/stylesheets/activities.css.scss [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
 // You can use Sass (SCSS) here: http://sass-lang.com/
+.participant-img {
+	width: 55%;
+	height: 55%;
+	margin: auto;
+	margin-top: 20px;
+}
+
+.list-group-item {
+	background-color: transparent;
```
Comment (frm) [inline app/assets/stylesheets/activities.css.scss]: Tabs?

**Review by frm — CHANGES_REQUESTED** [review]

File: app/views/activities/index.slim [hunk 2]
```diff
@@ -0,0 +1,19 @@
+= stylesheet_link_tag "stylsheets/glyphicons", media: "all"
```
Comment (dinocosta) [inline app/views/activities/index.slim]: The path in this stylesheet_link_tag is broken, besides the gplyhicons file is not present in the repository anymore so I think it's safe to remove this line :)

File: app/views/activities/index.slim [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+- (Date.new(@edition.begin_date.year, @edition.begin_date.month, (@edition.begin_date.day - 1))..Date.new(@edition.end_date.year, @edition.end_date.month, @edition.end_date.day)).each do |day|
```
Comment (dinocosta) [inline app/views/activities/index.slim]: Maybe you could create a variable in the controller to avoid having this range written here 👍

**Review by dinocosta — CHANGES_REQUESTED** [review]

File: app/controllers/activities_controller.rb [hunk 4]
```diff
@@ -4,6 +4,9 @@ class ActivitiesController < ApplicationController
   before_action :set_activity, only: [:show, :register, :deregister]
 
 	def index
+    begin_date = Date.new(@edition.begin_date.year, @edition.begin_date.month, (@edition.begin_date.day - 1))
```
Comment (frm) [inline app/controllers/activities_controller.rb:6]: what is this indentation?

**Review by frm — CHANGES_REQUESTED** [review]

File: app/controllers/activities_controller.rb (hunk 4 above)
Comment (joaofcosta) [inline app/controllers/activities_controller.rb]: 😂

**Review by frm — APPROVED** [review]

**Pull request merged**